chpl/chpl-api/src/test/java/gov/healthit/chpl/CHPLTestConfig.java

```diff
@@ -69,8 +70,10 @@
 @Configuration
 @Import({ChplTestCacheConfig.class})
 @EnableGlobalMethodSecurity(prePostEnabled = true)
-@PropertySource("classpath:/environment.test.properties")
-@EnableCaching
+@PropertySources({
+    @PropertySource("classpath:/environment.test.properties"),
+    @PropertySource("classpath:/lookup.test.properties"),
+})@EnableCaching
```

Comment:
Looks like this annotation is sneaking in behind the one above it. Nothing major, just looks strange and could be missed if looking over the class quickly.